README.md

```diff
@@ -0,0 +1,153 @@
+under the `BUNDLE_CAR` public const, for easier consumption by Rust code.
+
+Precompiled actor bundles may also be provided as release binaries in this repo,
+if requested by implementors.
+
+## Instructions for client implementations
+
+### Obtaining an actors bundle
+
+There are two options:
+
+1. Building from source.
+2. Downloading the precompiled release bundle from GitHub.
+
+Instructions to build from source (option 1):
+
+1. Clone the repo.
+2. Check out the relevant branch or tag (see Versioning section below).
+3. `cargo build` from the workspace root.
+4. Copy the CARv1 file generated the location printed in this log line:
+    ```
+   warning: bundle=/path/to/repo/target/debug/build/filecoin_canonical_actors_bundle-aef13b28a60e195b/out/bundle/bundle.car
+   ```
+   All output is printed as a warning only due to limitations in the Cargo build
+   script mechanisms.
+
+Both options are compatible with automation via scripts or CI pipelines.
+
+### Integrating an actors bundle
+
+This part is implementation-specific. Options include:
+
+1. Embedding the bundle's CARv1 bytes into the distribution's binary.
+2. Downloading CARv1 files on start (with some form of checksumming for added security).
+
+### Loading and using the actors bundle with ref-fvm
+
+Once the implementation has validated the authenticity of the bundle, it is
+expected to do the following:
+
+1. Import the CARv1 into the blockstore.
+2. Retain the root CID in memory, indexed by network version.
+3. Feed the root CID to ref-fvm's Machine constructor, to tell ref-fvm which
+   CodeCID maps to which built-in actor.
+
+### Multiple network version support
+
+Because every network version may be backed by different actor code,
+implementations should be ready to load multiple actor bundles and index them
+by network version.
+
+When instantiating the ref-fvm Machine, both the network version and the
+corresponding Manifest root CID must be passed.
+
+## Versioning
+
+A fair question is how crate versioning relates to the protocol concept of
+`ActorVersion`. We adopt a policy similar to specs-actors:
+
+- Major number in crate version correlates with `ActorVersion`.
+- We generally don't use minor versions; these are always set to `0`.
+- We strive for round major crate versions to denote the definitive release for
+  a given network upgrade. However, due to the inability to predict certain
+  aspects of software engineering, this is not a hard rule and further releases
+  may be made by bumping the patch number.
+
+Development versions will use qualifiers such as -rc (release candidate).
+
+As an example of application of this policy to a v10 actor version lineage:
+
+- Unstable development versions are referenced by commit hash.
+- Stable development versions are tagged as release candidates: 10.0.0-rc1, 10.0.0-rc2, etc.
+- Definitive release: 10.0.0.
+- Patched definitive release: 10.0.1.
+- Patched definitive release: 10.0.2.
+- Network upgrade goes live with 10.0.2.
```

Comment:
We will probably never have reason to use patch versions in FVM. They were usable in Go because changing the code didn't necessarily change its behaviour, but with compilation through WASM there'll be very few changes that don't change the output code and hence gas calculation at all. We bumped the minor version for any detectable behaviour change.